recipes/racket/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "6.12" %}
+
+package:
+  name: racket
+  version: {{version}}
+
+source:
+  fn: racket-{{ version }}.tar.gz
+  url: https://github.com/racket/racket/archive/v{{ version }}.tar.gz
+  sha256: c98f8d8f8dd62abdef9906adf7b1ce60b37c852b4111700c06bf0c2d290dd588
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
This has changed. Please see [these docs]( https://conda-forge.org/docs/meta.html#compilers ).